cpp/src/parquet/properties.h

```diff
@@ -757,6 +829,18 @@ class PARQUET_EXPORT WriterProperties {
     return false;
   }
 
+  bool bloom_filter_enabled() const {
+    // Note: we disallow enable bloom filter by default for now.
```

Comment:
Perhaps saying `We do not encourage enabling bloom filter for all columns. So default_column_properties_.bloom_filter_enabled is always false and cannot be altered by user. Thus we can safely skip checking it here.`